scripts/test.ipynb

```diff
@@ -2,133 +2,62 @@
  "cells": [
```

Comment:
Is this a Julia notebook? What does it do? I don't think that we want it in source code?